src/app/portlets/dot-edit-page/content/dot-edit-content.component.spec.ts

```diff
@@ -115,12 +115,16 @@ describe('DotEditContentComponent', () => {
                 {
                     provide: ActivatedRoute,
                     useValue: {
-                        data: Observable.of({
-                            content: {
-                                page: mockDotRenderedPage,
-                                state: mockDotPageState
+                        parent: {
```

Comment:
One endpoint, now everything comes from the top level of the routing which is the one that use the resolver.